sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/AmqpMessageWithLockToken.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus.implementation;
+
+import org.apache.qpid.proton.message.Message;
+
+import java.util.UUID;
+
+/**
+ * Holds loks token for AMQP message.
+ */
+public class AmqpMessageWithLockToken {
+    private final Message message;
```

Comment:
You have a class that represents this already? My PR overlaps with this. You can pull in my changes to reduce duplication